test/selenium/web_app_test.js

```diff
@@ -0,0 +1,48 @@
+
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+
+var HOST_URL = 'https://fxbox.github.io/app';
+
+var SetUpWebapp = require('./lib/setup_webapp.js');
+var webAppMainPage;
+var setUpWebapp;
+
+describe('UI app', function() {
+  var driver;
+  this.timeout(8000);
+
+  //const PASS = '12345678';
+
+  before(function() {
+    driver = new webdriver.Builder().
+      forBrowser('firefox').
+      build();
+  });
+
+  beforeEach(function() {
+    driver.get(HOST_URL);
+  });
+
+  after(function() {
+    driver.quit();
+  });
+     
+  describe('open the web app', function() {
+
+    beforeEach(function() {
+          setUpWebapp = new SetUpWebapp(driver);
+          webAppMainPage = setUpWebapp.getAppMainView();
+          return webAppMainPage;
+        });
+
+    it('should log in from web app', function() {
+      return webAppMainPage.connectToFoxBox().then((logInView) => 
+        logInView.loginSuccess(12345678));
```

Comment:
For reference @npark-mozilla  made some similar setup steps for the integration tests. One way to do is to [delete the sqlite files](https://github.com/fxbox/foxbox/blob/master/test/integration/lib/foxboxHelper.js#L33) and [start the foxbox process](https://github.com/fxbox/foxbox/blob/master/test/integration/lib/foxboxHelper.js#L40) one more time.